Readme.md

```diff
@@ -32,10 +32,33 @@ Many platforms require additional image processing beyond the EDK2 build.
 Any such steps should be documented (as a Readme.md), and any necessary helper
 scripts be contained, under said platform directory.
 
-Any contributions to this branch should be submitted via email to the
-edk2-devel mailing list with a subject prefix of `[platforms]`. See
-[Laszlo's excellent guide](https://github.com/tianocore/tianocore.github.io/wiki/Laszlo's-unkempt-git-guide-for-edk2-contributors-and-maintainers) for details
-on how to do this successfully.
+Any contributions to this branch should be submitted via GitHub Pull Request,

```

Comment:
This is a different process than edk2 where we only allowed **existing** patch series to continue using the mailing list after the pull request transition.

For simplicity/consistency, I'd prefer to follow that process and keep the PR related documentation in the [TianoCore wiki](https://github.com/tianocore/tianocore.github.io/wiki/EDK-II-Development-Process) relevant in this repo as well.

Reply:
Good point: I've deleted the section on email-based reviews.